corehq/apps/accounting/static/accounting/js/pricing_table.js

```diff
@@ -19,39 +31,57 @@ hqDefine('accounting/js/pricing_table', function () {
         self.selectCurrentPlan = function () {
             self.selected_edition(self.currentEdition);
         };
+        self.capitalizeString = function (s) {
```

Comment:
There's a version of this in main.js: [capitalize](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/hqwebapp/static/hqwebapp/js/main.js#L347)